hack/teardown-e2e-kind-cluster.sh

```diff
@@ -7,3 +7,8 @@ if ! command -v kind &> /dev/null; then
 fi
 
 kind delete cluster
+if [[ "${INTERFACES_SWITCHER}" == "system-service" ]];then
```

Comment:
This will not work if you use `--device-netns-switcher`

Reply:
done, can you have a look.